packages/reactive-dict/reactive-dict.js

```diff
@@ -124,6 +139,47 @@ _.extend(ReactiveDict.prototype, {
     if (_.has(self.keys, key)) oldValue = parse(self.keys[key]);
     return EJSON.equals(oldValue, value);
   },
+  clear: function(key){
+    var self = this;
+    self.set(key, null);
+    return true;
+  },
+  toggle: function(key){
+    var self = this;
+
+    // toggle currently only works on boolean values
```

Comment:
I'm pretty sure this ends up with the exact same result as `! Session.get('foo')`